{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/cli.py

```diff
@@ -0,0 +1,125 @@
+# =============================================================================
+# Make model specifications
+#
+# click application that takes a template model specification file
+# and locations for which to create model specs and uses jinja2 to
+# render model specs with the correct location parameters plugged in.
+#
+# It will look for the model spec template in "model_spec.in" in
+# the directory "src/<project_name>/model_specification". Specify multiple
+# locations in a file called "locations.txt" in the same directory. You
+# can create a single location using the "-s" option. Using the single
+# location option will override an existing locations file.
+#
+# The application will look for the model spec and locations files based
+# on the python environment that is active and these files don't need
+# to be specified if the default names and location are used.
+# =============================================================================
+import re
+import click
+from pathlib import Path
+from jinja2 import Template
+from collections import namedtuple
+
+from typing import List
+
+
+PROJ_NAME='{{cookiecutter.package_name}}'
+MODEL_SPEC_DIR=Path(__file__).parent.parent / 'model_specifications'
+DEFAULT_TEMPLATE_FILE=MODEL_SPEC_DIR / 'model_spec.in'
+DEFAULT_LOCATIONS_FILE=MODEL_SPEC_DIR / 'locations.txt'
+Location = namedtuple('Location', ['loc_proper', 'loc_sanitized'])
+
+
+COMMENT_CHAR='#'
```

Comment:
These vars should be local to the function that uses them unless there's a reason for them to be global.